src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((item) => {
+    const child = item.querySelector('ul');
+
+    if (!child) {
+      return;
+    }
```

Comment:
This block of code is checking if the `child` is null and if so, it returns from the function. This is a good practice, however, it would be more efficient to use `continue` instead of `return` to skip to the next iteration of the loop.